scripts/index.js

```diff
@@ -0,0 +1,128 @@
+const state = {
+    tempCount: 70
+};
+
+
+const updateTemp = tempValue => {
+    const tempCountContainer = document.getElementById("tempValue");
+    tempCountContainer.textContent = tempValue;
+}
+// Wave 2 - Increase Temp Behavior
+const addNum = () => {
+    state.tempCount += 1;
+    updateTemp(tempValue);
+};
+// Wave 2- Decrease Temp Behavior
+const subNum = () => {
+    state.tempCount -= 1;
+    updateTemp(tempValue);
+
+
+
+
+
+
+// Wave 2 - Change Color of Temp
+const tempRanges = () => {
+    if (state.tempCount <= 49){
+        document.getElementById("tempValue").style.color = "#008080";
+    } else if (state.tempCount >= 50 && state.tempCount <= 59){
+        document.getElementById("tempValue").style.color = "#00FF00";
+    } else if (state.tempCount >= 60 && state.tempCount <= 69){
+        document.getElementById("tempValue").style.color = "#FFF933";
+    } else if (state.tempCount >= 70 && state.tempCount <= 79){
+        document.getElementById("tempValue").style.color = "#FF5A33";
+    } else (state.tempCount >= 80);{
+        document.getElementById("tempValue").style.color = "#F8422D";
+    }
+
+}
+
+// Wave 2 - Temperature Ranges Change Landscape
+const tempLandscapeRanges = () => {
+    if (state.tempCount <= 59){
```

Comment:
This logic is correct, butwe need to update the text content of the "landscape" element. Consider first selecting the element such as `landscape = document.getElementById("landscape")` and then updating the textContent in each conditional `landscape.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"`